Dockerfile

```diff
@@ -41,10 +37,15 @@ RUN pip freeze 2>/dev/null > requirements.installed \
         || ( echo "!! ERROR !! requirements.txt defined different packages or versions for installation" \
                 && exit 1 ) 1>&2
 
-COPY static /code/static
-COPY templates /code/templates
-COPY truewiki /code/truewiki
-RUN mkdir /data
+# NOTE this volume stores your pages be sure to link it and make it persistent. Don't forget to create the subfolders manually : /data/File, /data/Category, /data/Page, /data/Template
```

Comment:
These comments in Dockerfiles are not useful to me. I rather see that in the README, as that is where people are reading how to use stuff.